style.css

```diff
@@ -22,31 +24,86 @@ body {
   flex-grow: 0;
 }
 
-header {
-  height: 95%;
-  width: 95%;
-  border-radius: 0.25rem;
+.navbar {
   display: flex;
   justify-content: space-between;
   align-items: center;
-  background-color: white;
+  background-color: #396df2;
+  color: white;
 }
 
 .Logo {
-  width: fit-content;
-  font-style: normal;
-  color: #6070ff;
-  font-size: 18px;
-  line-height: 20px;
-  text-align: center;
-  font-weight: 700;
-  margin: 10px 12px;
+  font-size: 1.5rem;
+  margin: 0.5rem;
+}
+
+.navbar-links ul {
+  margin: 0;
+  padding: 0;
+  display: flex;
+}
+
+.navbar-links li {
+  list-style: none;
+}
+
+.navbar-links li a {
   text-decoration: none;
+  color: white;
+  padding: 1rem;
+  display: block;
+}
+
+.navbar-links li a:hover {
+  background-color: #11256f12;
+}
+
+.toogle-button {
+  position: absolute;
+  top: 0.75rem;
+  right: 1rem;
+  display: none;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 30px;
+  height: 21px;
 }
 
-.Navbar {
-  width: 18px;
-  height: 12px;
+.toogle-button .bar {
+  height: 3px;
+  width: 100%;
+  background-color: white;
+  border-radius: 10px;
+}
+
+.overlay {
+  display: none;
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  z-index: 3;
+  background-color: #6070ff;
+  backdrop-filter: blur(3px);
```

Comment:
- [ ] Unfortunately, the backdrop filter you added didn't make the background of the mobile menu blurry as it should be, Please fix that to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://github.com/amiralda/amiral-portfolio/assets/68796232/f18b0dbc-dc2c-458e-950a-c806ef7c4fa6)  | ![image](https://github.com/amiralda/amiral-portfolio/assets/68796232/d6f478d0-658b-4094-be0e-597792647ff2)  |